qiskit/providers/ibmq/ibmqbackendservice.py

```diff
@@ -275,6 +255,76 @@ def jobs(
 
         return job_list
 
+    def _build_statuses_db_filter(self, statuses: List[Union[JobStatus, str]]) -> Dict[str, Any]:
+        """Return the db filter used to search for jobs that match any of the given statuses.
+
+        Return the db filter to use when querying the api for jobs that match any
+        of the given list of statuses.
+
+        Returns:
+            The status db filter used to query the api when searching for jobs that
+            match any of the given list of statuses.
+
+        Raises:
+            IBMQBackendError: If any of the status values is not recognized.
+        """
+        statuses_filter = {'or': []}  # type: Dict[str, List[Dict[str, Any]]]
```

Comment:
does this mean if the user has `or` specified in `db_filter` it'd essentially be discarded? I don't think we want that.